src/reynir_correct/errtokenizer.py

```diff
@@ -2447,6 +2579,11 @@ def late_fix_merges(
                     suggest=token.txt,
                 ),
             )
+        # Remove all annotations and revert corrections for tokens in wordlist
+        if token.original and token.original.strip() in ignore_wordlist:
```

Comment:
This uses exact-case comparison - is that intended? (I can see that it could be, just wanted to be sure.)

Reply:
Good point. For now, it only checks for the case present in the whitelist. I added a note detailing this, the code can then be changed according to updated user needs.